supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -547,7 +548,7 @@ There is no functional difference between the first server that was installed an
 [discrete]
 [[io]]
 ==== image:images/yes.png[yes] io subsystem (noun)
-*Description*: In Red Hat JBoss Enterprise Application Platform, the "io" subsystem is used to define workers and buffer pools used by other subsystems. In general text, write in lowercase as one word. Use "IO subsystem" when referring to the io subsystem in titles and headings.
+*Description*: In Red Hat JBoss Enterprise Application Platform, the _`io` subsystem_ is used to define workers and buffer pools used by other subsystems. In general text, write "io" in lowercase as one word. Use "IO subsystem" when referring to the `io` subsystem in titles and headings.
```

Comment:
_`io` subsystem_ is not showing up with correct formatting for me. Double check this.

Reply:
Fixed.